php/api/endpoint/class-audit.php

```diff
@@ -269,23 +265,15 @@ public function rest_results_get( $rest_post, $field_name, $request ) {
 				continue;
 			}
 
-			$results[ $standard ] = $meta;
+			if ( ! empty( $meta['full'] ) ) {
```

Comment:
The legacy data will never exist if we start from scratch. Not sure we need to do this or unset the `details` below. Once the Go services are updated the stored data will be correct.

Reply:
This is minimal impact to store the references. We have not discussed another way yet.  A lot more will have to change in how the reports get fetched if we change this structure.